pkg/build/builder/util.go

```diff
@@ -59,3 +59,29 @@ func getDockerNetworkMode() stiapi.DockerNetworkMode {
 	}
 	return ""
 }
+
+// MergeEnv will take an existing environment and merge it with a new set of
```

Comment:
i am ok leaving as it is now :) thanks for looking into it.
On Dec 9, 2015 10:50 PM, "Cesar Wong" <notifications@github.com> wrote:

> In pkg/build/builder/util.go
> <https://github.com/openshift/origin/pull/6178#discussion_r47157467>:
>
> > @@ -59,3 +59,29 @@ func getDockerNetworkMode() stiapi.DockerNetworkMode {
> >  	}
> >  	return ""
> >  }
> > +
> > +// MergeEnv will take an existing environment and merge it with a new set of
>
> @mfojtik <https://github.com/mfojtik> I looked into merging this function
> with the one in the controller code. However, this one deals with vars as
> strings whereas the other one looks at kapi.EnvVars. It seemed that getting
> the code reuse would require more code to be written to convert from one to
> the other. If you still feel strongly about merging them I can try it,
> otherwise I'll leave it alone.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6178/files#r47157467>.
>